test/common/instance_api.test.lua

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env tarantool
+
+local socket = require('socket')
+local tap = require('tap')
+package.cpath = './?.so;' .. package.cpath
+local memcached = require('memcached')
+local test = tap.test('memcached instance api')
+
+local function is_port_open(port)
+    local sock, _ = socket.tcp_connect('127.0.0.1', port)
```

Comment:
Nit: why do you drop the error message here and don't display it? AFAIC, it could benefit test failure diagnostics.